specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,52 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily understand the differences that make up the multiple **SKU Price ID**s under a given [**SKU ID**](#skuid).
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuPriceDetails_ column MUST be shared across all resources having the same _SkuPriceId_.
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are already captured in the [_SkuDetails_](#skudetails) column.
```

Comment:
This presents a significant usability problem for practitioners because they need to intimately understand the difference between SkuId and SkuPriceId, which we don't actually define well in the spec. Splitting attributes across two JSON objects will make it harder for people to find the details they care about because in some cases it may be in one and others in the other. From a usability perspective, this sounds like a horrible idea. 

As a product owner, I would never do this without a usability test. I don't think we should be doing it here either. We should start with one and split it out if there's a reason that doesn't work. If we add 2, then we're creating a usability problem upfront that will be incredibly hard to pull back. It's always easier to add something than remove something, so starting with a single SKU details column is just simpler and safer all-around.